.github/workflows/build.yml

```diff
@@ -10,11 +10,9 @@ jobs:
     steps:
     - uses: actions/checkout@v2
     - name: Set up Ruby 2.6
-      uses: actions/setup-ruby@v1
+      uses: ruby/setup-ruby@v1
       with:
-        ruby-version: 2.6.x
+        ruby-version: 2.6
```

Comment:
Looks like these can be dropped now and it will read the `.ruby-version` https://github.com/ruby/setup-ruby#supported-version-syntax